components/k8s-model-server/images/releaser/components/workflows.libsonnet

```diff
@@ -88,9 +161,12 @@
                   "/usr/local/bin/checkout.sh",
                 ],
                 args: [
-                  srcDir,
+                  srcRootDir,
                 ],
-                env: prow_env,
+                env: prow_env + [{
+                  "name": "EXTRA_REPOS",
+                  "value": "tensorflow/k8s@HEAD;kubeflow/testing@HEAD:12",
```

Comment:
Did you mean to pin to kubeflow/testing#12? has that been submitted yet?

Reply:
removing pin